docs/tutorials/tutorial-update-data.md

```diff
@@ -118,22 +131,24 @@ dsql> select * from "updates-tutorial";
 Retrieved 6 rows in 0.02s.
 ```
 
-## Append to the data
+## Append data
 
-Let's try another way of appending data.
+Now you append data to the datasource without changing the existing data.
+Use the ingestion spec located at `quickstart/tutorial/updates-append-index2.json`.
 
-The `quickstart/tutorial/updates-append-index2.json` task spec reads input from `quickstart/tutorial/updates-data4.json` and will append its data to the `updates-tutorial` datasource. Note that `appendToExisting` is set to `true` in this spec.
+The spec directs Druid to ingest data from `quickstart/tutorial/updates-data4.json` and append it to the `updates-tutorial` datasource. The property `appendToExisting` is set to `true` in this spec.
 
-Let's submit that task:
+Submit the task:
 
 ```bash
 bin/post-index-task --file quickstart/tutorial/updates-append-index2.json --url http://localhost:8081
 ```
 
-When the new data is loaded, we can see two additional rows after "octopus". Note that the new "bear" row with number 222 has not been rolled up with the existing bear-111 row, because the new data is held in a separate segment.
+When the task completes, query the data again. Two additional rows are added after `octopus`.
+Druid does not roll up the new `bear` row with the existing `bear` row because it stored the new data in a separate segment.
```

Comment:
```suggestion
Druid doesn't roll up the new `bear` row with the existing `bear` row because it stored the new data in a separate segment.
```
Google recommends using contractions:
https://developers.google.com/style/contractions